modules/ocm_cluster_rosa_classic/operator_roles/cloud_credential_role.tf

```diff
@@ -0,0 +1,32 @@
+resource "aws_iam_role" "cloud-credential_role" {
+  name = "${var.operator_role_prefix}-openshift-cloud-credential-operator-cloud-c"
```

Comment:
"A name for the IAM role, up to 64 characters in length." [1](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html)

What we do the in the CLI in setup the whole name and truncate it to first 64. The ARN does not have this restriction so it will have the whole string

Should that truncate operation be needed here?

Reply:
My plan is to use only the operator_role_prefix and send the name explicitly. 
This will be part of the next PR